engine.py

```diff
@@ -112,18 +112,22 @@ def get_knight_moves(self, r, c, moves):
 
 	def get_king_moves(self, r, c, moves):
  		##TODO
+		rows = (r-1, r, r+1)
+		columns = (c+1, c, c-1)
 		if self.light_to_move: # if it's light's turn to move
+			box = ("  ", "d")
```

Comment:
Rename tuple from _box_ to  _available_squares_, it is more descriptive.
first element of tuple should be only one space " " not "  ", so that later on line 122 and 126 you only have to do one check for availble_squares.
This line should be;

available_squares = (" ", "d")  #if square is empty or has a dark piece

include the comment